.pre-commit-config.yaml

```
@@ -20,6 +20,12 @@ repos:
     hooks:
       - id: check-dependabot
       - id: check-github-workflows
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.2.4
```

```suggestion
    rev: v2.3.0
```
I believe `2.3.0` is the latest release.
https://github.com/codespell-project/codespell/releases/tag/v2.3.0